daemon/graphdriver/windows/windows.go

```diff
@@ -47,6 +48,10 @@ type Driver struct {
 	active map[string]int
 }
 
+func ensureDriverInterface(d *Driver) graphdriver.DiffGetterDriver {
```

Comment:
I don't think this is used anywhere, is it?

We might not need to do this either, do we?